core/trino-main/src/main/java/io/trino/sql/planner/iterative/rule/InlineProjections.java

```diff
@@ -122,6 +122,10 @@ static Optional<ProjectNode> inlineProjections(PlannerContext plannerContext, Pr
         for (Map.Entry<Symbol, Expression> assignment : child.getAssignments().entrySet()) {
```

Comment:
Please follow https://github.com/trinodb/trino/blob/master/.github/DEVELOPMENT.md#format-git-commit-messages and improve the commit message

Reply:
Thanks for the tips. Improved commit message.